flink-metrics/flink-metrics-influxdb/src/test/java/org/apache/flink/metrics/influxdb/MeasurementInfoProviderTest.java

```diff
@@ -56,9 +78,26 @@ public void simpleTestGetMetricInfo() {
         assertEquals(
                 String.join("" + MeasurementInfoProvider.SCOPE_SEPARATOR, logicalScope, metricName),
                 info.getName());
-        assertThat(info.getTags(), hasEntry("A", "a"));
-        assertThat(info.getTags(), hasEntry("B", "b"));
-        assertThat(info.getTags(), hasEntry("C", "c"));
+        assertThat(info.getTags(), hasEntry(KEY1, VALUE1));
+        assertThat(info.getTags(), hasEntry(KEY2, VALUE2));
+        assertThat(info.getTags(), hasEntry(KEY3, VALUE3));
```

Comment:
It is not intuitive why this actually works; it only does because the implementation is removing characters instead of replacing them with others.

Reply:
ok